lib/pip.js

```diff
@@ -2,6 +2,16 @@ const fse = require('fs-extra');
 const path = require('path');
 const {spawnSync} = require('child_process');
 
+// Detection of Windows Subsystem for Linux
+try {
+  const isWsl = require('is-wsl');
+} catch (e) {
```

Comment:
Is there a good reason to expect the `is-wsl` module not to be importable? It's in the dependencies and doesn't look like it'd crash on other platforms for any reason (and I just tested it on linux without import errors).